### PR TITLE
dash/project-demo

### PR DESCRIPTION
**Pull request by karolkolodziej**
Added the project management demo.

### CODE REVIEW

File: samples/dashboards/demo/project-management/demo.css [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+    stroke: var(--gray);
+}
+
+#dashboard-chart-1 .highcharts-point.highcharts-color-0 {
+    fill: var(--gray);
+    stroke: var(--gray);
+}
+
+#dashboard-kpi-3 .highcharts-dashboards-component .highcharts-point.highcharts-color-0 {
+    fill: black;
```
Comment (sebastianbochan) [inline samples/dashboards/demo/project-management/demo.css:45]: Please use hex instead.

**Review by Vazonik — APPROVED** [review]
Looks good! 🎉

File: samples/dashboards/demo/project-management/demo.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+<script src="https://code.highcharts.com/highcharts.js"></script>
+<script src="https://code.highcharts.com/dashboards/modules/dashboards-plugin.js"></script>
+<script src="https://code.highcharts.com/gantt/modules/gantt.js"></script>
+<script src="https://code.highcharts.com/modules/accessibility.js"></script>
+
+<div id="container"></div>
+
+<p class="highcharts-description">
+    This demo shows how to use the Highcharts dashboards to create a project
+    management dashboard. <br>
```
Comment (TorsteinHonsi) [inline samples/dashboards/demo/project-management/demo.html:11]: Prefer using paragraphs, not single breaks. https://github.com/highcharts/highcharts/blob/master/repo-guidelines.md#1-avoid-line-breaks-without-paragraphs

File: samples/dashboards/demo/project-management/demo.js [hunk 3]
```diff
@@ -0,0 +1,455 @@
...
+        linkedValueTo: {
+            enabled: false
+        }
+    }, {
+        cell: 'dashboard-kpi-4',
+        type: 'Highcharts',
+        title: 'Tasks by status',
+        chartOptions: {
+            xAxis: {
+                categories: ['Done', 'To Do', 'In Progress', 'Blocked']
```
Comment (TorsteinHonsi) [inline samples/dashboards/demo/project-management/demo.js:175]: Pie series don't use `xAxis` 🤔

**Review by TorsteinHonsi — COMMENTED** [review]
Nice work! I have some comments inline, and some design related.

1. The dashboard title ("Current sprint") has its own card, which looks a bit weird. I think it would look better if it was either outside the cell structure, or the cell background and shadow was removed. I see we have solved this in different ways in the other demos.

2. The #current-sprint div has its own background color, making the background of the top row darker than the rest of the dashboard. Is that intended? It looks a bit unmotivated.

3. On the mobile, we use almost all the screen estate above the fold to show two numbers; completed tasks and incomplete tasks. Maybe it would be better to shrink the two cells to 50% and adjust the font sizes accordingly?

Skjermbilde 2023-11-29 kl  10 51 19

**Review by sebastianbochan — CHANGES_REQUESTED** [review]

File: samples/dashboards/demo/project-management/demo.js [hunk 4]
```diff
@@ -0,0 +1,455 @@
...
+        title: 'Timeline',
+        chartConstructor: 'ganttChart',
+        chartOptions: {
+            chart: {
+                marginLeft: 10
+            },
+            xAxis: [{
+                plotLines,
+                dateTimeLabelFormats: {
+                    day: '%e<br><span style="opacity: 0.5; font-size: 0.7em">%a</span>'
```
[the remark this answers is not in the document]
Comment (TorsteinHonsi) [reply samples/dashboards/demo/project-management/demo.js:272]: > use `<div>`

I don't think the SVG TextBuilder supports that

[the remark this answers is not in the document]
Comment (TorsteinHonsi) [reply samples/dashboards/demo/project-management/demo.js:272]: It doesn't: https://jsfiddle.net/highcharts/df0v6sn4/

File: samples/dashboards/demo/project-management/demo.js (hunk 3 above)
Comment (karolkolodziej) [reply samples/dashboards/demo/project-management/demo.js:175]: leftover 😅

**Comment by karolkolodziej** [comment]
3. It Seems valid but responsive options don't support the height. I added that to the backlog and we will change the demo after it is implemented.

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
Overall it looks great!

The contrast between the series types is too low (they are all very dark), but I know other team members are working on that. 

One thing that I think should be fixed is the tooltip for the Gantt chart. Both start and end date shows "Week from...", which is not very specific. I assume this happens because both dates are defined on the Monday midnight. There are two ways of resolving it; either set an explicit time format for the tooltip, or change the dates. Actually in this case I think it would make sense to reduce the length of the tasks. What about starting Monday 9:00 and ending them Friday at 17:00? That is a more realisting work week, and will probably resolve the tooltip.

**Comment by TorsteinHonsi** [comment]
Thanks! Not a big deal, but wouldn't it be more realistic if the tasks end on Friday 17:00 instead of the next Monday 17:00? Like fill out the work week and keep the weekend off?

File: samples/dashboards/demo/project-management/demo.css [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+.highcharts-plot-line {
+    stroke-width: 3px;
+    stroke: var(--highcharts-neutral-color-100);
+}
+
+.highcharts-plot-line-label {
+    fill: var(--highcharts-neutral-color-100);
+    font-size: 1em;
+    font-weight: bold;
+}
```
Comment (sebastianbochan) [inline samples/dashboards/demo/project-management/demo.css:123]: So many duplicates in declarations, you can optimize it.

**Comment by nasin** [comment]
Hi, I think it looks great. 
If possible, can you increase the margin a bit around the component header like in the screenshot?
image

Also, I'm not sure if the "done" color (that darker green color) should be used for the timeline, unless all those tasks are "done." If the timeline is just a timeline, I would use the almost black color like you have in the column chart. And maybe make the "today" line a distinct color from everything else so it stands out more.
image

**Review by marvin19 — APPROVED** [review]

**Review by TorsteinHonsi — CHANGES_REQUESTED** [review]
We need to do something with the colors of the area chart. I know we did this for a11y compliance, but according to common sense it is not really usable when the legend color is brown and the area color is yellow. Yes, the line color is brown, but that's barely distinguishable from the other lines.

As far as I know, the a11y aspect is covered since we have text directly on the area, so can we just use normal colors?

Skjermbilde 2023-12-11 kl  12 46 32

**Comment by karolkolodziej** [comment]
@TorsteinHonsi I'm ok with that- it seems more logical 😉

**Review by TorsteinHonsi — APPROVED** [review]

**Pull request merged**